enzyme/Enzyme/TypeAnalysis/TypeTree.h

```diff
@@ -650,6 +652,8 @@ class TypeTree : public std::enable_shared_from_this<TypeTree> {
                 chunk = 8;
               } else if (flt->isHalfTy()) {
                 chunk = 2;
+              } else if (flt->isX86_FP80Ty()) {
```

Comment:
We might as well here (within isFloat) simplify this to `chunk = dl.getTypeSizeInBits(flt) / 8;`

Reply:
Thanks, changed in [24c8098](https://github.com/EnzymeAD/Enzyme/pull/1604/commits/24c8098f99d83af45a0d681a44d9ec9411d40be3)